packages/design-system-tokens/src/lib/conversions.ts

```diff
@@ -0,0 +1,33 @@
+const PIXELS_PER_REM = 16;
+const PIXELS_PER_EX = 8.5625;
```

Comment:
Curious about where this unit comes from? It's my understanding that `ex` units vary based on the font-size, so it seems an `ex` unit in one theme may be different in another.

Reply:
Shoot, I think you're right. It really just comes from working backwards from the values we had to have in Figma. The designs use static pixel values for these, so if it has a constant here to convert it back into one `ex` value, it just means that in the browser it might have slightly different widths than the designs depending on what theme you're using. In my mind, it's not that bad, but I should definitely explain this in the comment if nothing else.